object-cache.php

```diff
@@ -273,31 +278,132 @@ function delete( $id, $group = 'default' ) {
 		return $result;
 	}
 
+	// Gets number from all default servers, replicating if needed
+	function get_max_flush_number( $group ) {
+		$key = $this->key( $this->flush_key, $group );
+
+		$values = array();
+		$size = 19;
```

Comment:
Prolly worth a comment. :) There's a chance it's 0 on some nodes, so this is the worst case I guess. Then again, I don't think it would be any performance impact if you called the `get_data_size()` function for all of these hardcoded 19s.

Reply:
@aidvu If a `get` returns an empty value, the `not_in_memcache` operation log will contain a size of `null`. Was there another case where we would get value of a different size?

Reply:
I think it will do `strlen( serialize( null ) )`, so won't be `null` really. But doesn't matter, I'm fine with 19 as long as there's a comment. Or name the variable `flush_key_size`.

Reply:
I copied the `null` size from line 450 where `get()` logs `not_in_memcache`. Same enough to leave as-is?

Reply:
I see what you mean. Sure. But do add a comment explaining the size 19. It's obvious to us, but I can bet you it will take someone else a lot longer to figure it out. 😄